docs/sensors/lidar_3d/velodyne_puck hi_res.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Velodyne  Hi-Res
+sidebar_position: 3
+---
+
+import SoftwareBringup from "/components/software_sensor_iso_supported.mdx";
+import Support from "/components/support.mdx";
+
+<center>
+  <img
+    src="/img/sensor_images/sensor_velodyne_puck_1.jpg"
+    width="400"
+  />
+</center>
+
+## Item Numbers
+
+| Description                              |                         CPR Item                         | Sales Kit | Installation Kit, With Fasteners |
+| :--------------------------------------- | :------------------------------------------------------: | :-------: | :------------------------------: |
+| Velodyne Puck sensor                     | [013688](/assets/pdf/clearpath_robotics_029029-TDS1.pdf) |     —     |                —                 |
+| Velodyne Puck sensor, with M12 connector | [029029](/assets/pdf/clearpath_robotics_029029-TDS1.pdf) |  029032   |              029031              |
+
+---
+
+## Specifications
+
+| Description                 | Value                |
+| :-------------------------- | :------------------- |
+| Mass                        | 830 g                |
+| Voltage, Minimum            | 9 V                  |
+| Voltage, Maximum            | 18 V                 |
+| Voltage, Sensor             | 12 V                 |
+| Power, Average              | 8 W                  |
+| Data                        | Ethernet, 100 Mbit/s |
+| Range, Minimum              |                      |
```

Comment:
Is the 'Minimum' unknown or N/A?

Reply:
Velodyne does not state a minimum in their datasheets or manuals.
The ROS driver disregards data that is closer than 0.9 metres:
https://github.com/ros-drivers/velodyne/blob/7c2f199f552ecb9f58a3c30734681042b1b99f82/velodyne_pointcloud/launch/transform_nodelet.launch#L11

Should we add 0.9 metres to our table's cell?

Reply:
I think we should add something to the cell. I'm ok with any of:

- N/A
- 0.9 m

Reply:
I updated our 4 Velodyne pages to say to 0.9 m.